apps/docs/src/app/core/component-docs/dialog/dialog-docs.module.ts

```diff
@@ -47,7 +48,8 @@ const routes: Routes = [
         DialogPositionExampleComponent,
         TemplateBasedDialogExampleComponent,
         ComponentBasedDialogExampleComponent,
-        DialogBackdropContainerExampleComponent
+        DialogBackdropContainerExampleComponent,
+        DialogObjectComponent
```

Comment:
Naming

Reply:
@salarenko in order to avoid confusion/further questions please provide more meaningful comments

Reply:
If all example components have "Example" suffix, new example components also should have "Example" suffix in component name.